packages/expo-cli/src/commands/start.js

```diff
@@ -136,7 +135,10 @@ async function validateDependenciesVersions(projectDir, exp, pkg) {
       );
     });
     log.warn(
-      'Your project may not work correctly until you install the correct versions of the packages.'
+      'Your project may not work correctly until you install the correct versions of the packages.\n' +
+        `To install the correct versions of these packages, please run: ${chalk.inverse(
+          'expo install [package-name ...]'
```

Comment:
Just thought of a nice extra: it would be cool if it asked "would you like to install them now?" (unless in `nonInteractive` mode) and if you chose "y", would then run the `install` command and continue after it's finished.

Reply:
Yeah, it would be cool, but it might be annoying for people that do want to use incompatible versions. It might be the case when installing an older or newer (incompatible in versions range, but compatible in code) version may be a workaround for an issue. 🤔